Sources/TuistGenerator/Settings/DefaultSettingsProvider.swift

```diff
@@ -134,6 +134,11 @@ public final class DefaultSettingsProvider: DefaultSettingsProviding {
                     .filter { $0.key > xcodeVersion }
                     .values.flatMap { $0 }.contains(key)
             }
+        case .excluding(let base, let excludedKeys):
+            let baseFilter = try self.createFilter(defaultSettings: base, essentialKeys: essentialKeys, newXcodeKeys: newXcodeKeys)
+            return { key, value in
+                !excludedKeys.contains(key) && baseFilter(key, value)
+            }
```

Comment:
Note this doesn't take into account the recommended case - it should be easier to implement with the proposed enum changes.